content/open-source/build-tools/apko/faq.md

```diff
@@ -18,20 +18,19 @@ toc: true
 ---
 
 ## Do I need to understand apko to use Chainguard Images?
-No. Chainguard built [apko](https://github.com/chainguard-dev/apko) as part of its open source tooling under the [Wolfi](/open-source/wolfi) operating system. While you can check out the [project on GitHub](https://github.com/chainguard-dev/apko) and learn more, it's not a prerequisite for using or [working with Chainguard Images](/chainguard/chainguard-images/working-with-images).
+No. Chainguard built [apko](https://github.com/chainguard-dev/apko) as part of its open source tooling around the [Wolfi](/open-source/wolfi) operating system. While you can check out the [project on GitHub](https://github.com/chainguard-dev/apko) and learn more, it's not a prerequisite for using or [working with Chainguard Images](/chainguard/chainguard-images/working-with-images).
 
-## How are melange packages defined?
-melange apks are defined declaratively using a YAML file.
+## How are apko images defined?
+apko images are defined declaratively using a YAML file. It was designed this way to facilitate reproducible builds - run apko twice, and you'll get the same output.
 
-## Are melange apks compatible with Alpine?
-Yes, melange apks are built to be compatible with apk-based systems including Alpine.
+## Does apko provide SBOMs?
+Yes, apko builds include high-quality SBOMs (software bills of materials) for all builds. This is a key feature of the tooling that Chainguard has developed to ensure that users can trust the software they are running.
```

Comment:
nbd but you could link to the SBOM explainer (https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) if you want

Reply:
good idea!